sale_order_revision/models/sale_order.py

```diff
@@ -42,3 +42,10 @@ def action_view_revisions(self):
             "default_current_revision_id": self.id,
         }
         return result
+
+    def action_cancel_create_revision(self):
+        for sale in self:
+            sale.action_cancel()
+            action = sale.create_revision()
+        if len(self) == 1:
```

Comment:
@dreispt You should always return something in action buttons. This prevent to be overridden

Reply:
Done!